app/views/plans/goals.html.erb

```diff
@@ -7,89 +7,79 @@
-                </td>
-              </tr>
-
-              <% assessment_technical_area.assessment_indicators.drop(1).each do |assessment_indicator| %>
-                <tr class="technical-area-<%= assessment_technical_area.id %>"
-                    data-controller="score-and-goal">
-                  <td class="border-right">
+      <div class="col border-bottom">
+        <div class="d-none d-sm-none d-md-block">
+          <div class="row border-top no-gutters">
+            <div class="col-md-auto w-220">
+              <h3>Technical Areas</h3>
+            </div>
+            <div class="header col-md">
+              <h3>Indicators</h3>
+            </div>
+            <div class="bordeheader col-md w-180 d-flex flex-row justify-content-md-around">
+              <h3 class="text-center">Score</h3>
+              <h3 class="text-center">Goal</h3>
+            </div>
+          </div>
+        </div>
+        <% @plan.assessment_technical_areas.each do |assessment_technical_area| %>
+          <div class="row no-gutters technical-area-<%= assessment_technical_area.id %>" data-controller="score-and-goal">
+            <div class="technical-area w-220 col-sm-12 col-md-auto">
+              <strong><%= assessment_technical_area.text %></strong>
+            </div>
+            <div class="col-sm-12 col-md">
+              <% assessment_technical_area.assessment_indicators.each do |assessment_indicator| %>
+                <div class="<%= counter % 2 == 0 ? 'odd' : '' %> row no-gutters">
+                  <% counter += 1 %>
+                  <div class="indicators col-sm-12 col-md">
                     <strong><%= abbrev_from_named_id assessment_indicator.named_id %></strong>
                     <%= assessment_indicator.text %>
-                  </td>
-                  <td>
-                    <%= render "validated_field", named_id: assessment_indicator.named_id, value: @plan.score_value_for(assessment_indicator: assessment_indicator), is_goal: false %>
-                  </td>
-                  <td>
-                    <%= render "validated_field", named_id: assessment_indicator.named_id, value: @plan.calculate_goal_value_for(assessment_indicator: assessment_indicator), is_goal: true %>
-                  </td>
-                </tr>
+                  </div>
+                  <div class="score-goal w-180 col-sm-12 d-flex flex-row justify-content-between">
+                    <div class="col d-flex flex-row p-0 justify-content-start">
+                      <div class="d-md-none col-sm-auto label">
+                        <strong>Score</strong>
+                      </div>
+                      <%= render "validated_field", named_id: assessment_indicator.named_id, value: @plan.score_value_for(assessment_indicator: assessment_indicator), is_goal: false %>
+                    </div>
+                    <div class="col d-flex flex-row p-o justify-content-start">
```

Comment:
is this `p-o` a pee-letter-oh and not a pee-zero

Reply:
fixed